docs/guide/gui/button.md

```diff
@@ -1,30 +1,30 @@
 # UIButton
 
-[UIButton](/api/classes/UIButton.md) 继承自 [UIInteractive](/api/classes/UIInteractive)（可交互组件）, 实现基本的按钮功能。 按钮组件内部封装有 `UIImage` 用于显示背景图，集成自可交互组件 `UIInteractive`，使用内置的 `UITransform` 描述的方形区域作为感应热区。
+[UIButton](/api/classes/UIButton.md) inherits from [UIInteractive](/api/classes/UIInteractive)(an interactive component), implementing basic button functions. The button component is internally encapsulated with `UIImage` for displaying background images, integrated with the interactive component `UIInteractive`, and uses the square area described by the built-in `UITransform` as the sensing heat zone.
```

Comment:
sensing heat zone can use the hotspot to replace, I double-checked the use of hotspots in UI design guidelines, such as Ant design